### PR TITLE
html-forms - Marianne Cuevas

### PR DESCRIPTION
**Pull request by mariannecuevas**
Animation

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          <option value="option">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <div>
+        <label for="html">
+          <input type="radio" name="language" value="html"/> HTML
```
Comment (kurtyazdizadeh) [inline html-forms/index.html]: If you are using the `for` attribute on the `label` tag is there a matching attribute that needs to be on the `input` for it to work properly?

The user should be able to click on the text of the label to select the input as well.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      <div>
+        <label for="sign-up">
+          <input type="checkbox" name="more-info" checked="checked" />Sign me up!
+        </label>
+      </div>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <div>
+        <textarea rows="10" cols="32" id="comments"></textarea>
```
Comment (kurtyazdizadeh) [inline html-forms/index.html:44]: Hint: You did it properly here 😊 Just apply the right attribute to your other form elements and you'll be all set!

**Review by kurtyazdizadeh — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 1 above)
Comment (mariannecuevas) [reply html-forms/index.html]: There has to be an id attribute matching the for attribute. Can the value and id attributes have the same values?

Comment (kurtyazdizadeh) [reply html-forms/index.html]: good question - yes it's okay for those to be the same

**Review by kurtyazdizadeh — APPROVED** [review]
Approved 👍

**Pull request merged**